src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
-
+        int supplyTotal = 0;
+        int buyTotal = 0;
```

Comment:
The variables 'supplyTotal' and 'buyTotal' are descriptive enough, but the method could be refactored to avoid using these as accumulators directly within the file reading loop.